kubernetes/apps/mealie/mealie/values.yaml

```diff
@@ -48,19 +48,20 @@ service:
     controller: mealie
     ports:
       http:
-        port: 9000
+        port: &port 9000
 
-ingress:
+route:
   app:
-    enabled: true
-    className: internal
-    hosts:
-      - host: mealie.laboratory.casa
-        paths:
-          - path: /
-            service:
-              identifier: app
-              port: http
+    hostnames:
+      - mealie.laboratory.casa
+    parentRefs:
+      - name: internal
+        namespace: kube-system
+        sectionName: https
+    rules:
+      - backendRefs:
+          - identifier: app
+            port: *port
```

Comment:
This is from the BJW-S app template I use for most things, it has a [`route` resource](https://bjw-s-labs.github.io/helm-charts/docs/app-template/#__codelineno-0-58) you can use like this. By default it will do a: 

```
matches:
  - path:
      type: PathPrefix
      value: /
```

So you can omit that unless the app needs something different.